.github/workflows/binary.yaml

```diff
@@ -0,0 +1,81 @@
+name: Binaries
+
+on:
+  schedule:
+    - cron: '0 1 * * *' # Triggers the build at 1:00 UTC time
+
+jobs:
+  binary:
+    name: Create
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: macos-13
+            name: x86_64-apple-darwin
+            installable: .#
+          - os: macos-14
+            name: aarch64-apple-darwin
+            installable: .#
```

Comment:
Are we sure these are `amd64` and `arm64`?

Reply:
This is currently what we have in the `ocaml/dune` repository.